src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -10,6 +10,7 @@ import { UserService } from '../shared/user.service';
           i18n mat-raised-button>{{ planetType === 'center' ? 'Nation' : 'Community' }}</a>
       </span>
       <a routerLink="/feedback" i18n mat-raised-button>Feedback</a>
+      <a routerLink="configuration/update/{{configurationId}}" il8n mat-raised-button>Configuration</a>
```

Comment:
There will be only one configuration per planet so configuration id will not be required to pass